### PR TITLE
76X 25ns HLT round three

### PR DESCRIPTION
**Pull request by Martin-Grunewald**
76X 25ns HLT round three of 25ns HLT menu updates
Based on 760pre4
Includes previous PR #10836

### CODE REVIEW

File: Configuration/HLT/python/autoHLT.py [hunk 1]
```diff
@@ -3,9 +3,9 @@
 
 autoHLT = {
   'frozen25ns' : '25ns14e33_v3',
-  'relval25ns' : 'GRun',
+  'relval25ns' : '25ns14e33_v3'
```
Comment (perrotta) [inline Configuration/HLT/python/autoHLT.py]: Missing comma...

**Comment by Martin-Grunewald** [comment]
@perrotta 
thanks, fixed.

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
please test

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**